GameData/ROCapsules/PartConfigs/GeminiBDB/RP_GeminiBDB.cfg

```diff
@@ -0,0 +1,53 @@
+//Star 13E retro
+@PART[ROC-GeminiRetroBDB]:FOR[RealPlume]:NEEDS[SmokeScreen]
```

Comment:
Do not declare a :FOR pass that is not for our own mod.  Must use BEFORE or AFTER